develop/develop-images/build_enhancements.md

```diff
@@ -187,9 +187,9 @@ $ docker build --no-cache --progress=plain --secret id=mysecret,src=mysecret.txt
 > Please see [Build secrets and SSH forwarding in Docker 18.09](https://medium.com/@tonistiigi/build-secrets-and-ssh-forwarding-in-docker-18-09-ae8161d066)
 > for more information and examples.
 
-The `docker build` has a `--ssh` option to allow the Docker Engine to forward
-SSH agent connections. For more information on SSH agent, see the
-[OpenSSH man page](https://man.openbsd.org/ssh-agent).
+The `docker build` has an `--ssh` option to allow the Docker Engine to forward
+SSH agent connections. Passing `--ssh default` is equivilent to `--ssh default=$SSH_AUTH_SOCK`.
```

Comment:
At this point the reader has no idea what `--ssh default` means. Possibly add it to the end of this section and explain the shortcut after explaining the format.

Reply:
I rewrote this section somewhat to be more step-by-step. See what you think @tonistiigi .

I still don't know what `default` means in `--ssh default=$SSH_AUTH_SOCK` - like, would you ever do `--ssh secondary=$SSH_AUTH_SOCK`, and if so, why? Maybe someone else can chime in to add that explanation.

Reply:
Yes, `id` is for namespacing sockets like you show in the example.

The flag also accepts keys directly.